python/amici/petab_objective.py

```diff
@@ -90,10 +90,8 @@ def simulate_petab(
     :return:
         Dictionary of
 
-        * cost function value (LLH),
-        * const function sensitivity w.r.t. parameters (SLLH),
-          (**NOTE**: Sensitivities are computed for the scaled parameters)
-        * list of `ReturnData` (RDATAS),
+        * cost function value (``LLH``),
+        * list of :class:`amici.ReturnData` (``RDATAS``),
```

Comment:
Not linked. Looking at the type hints in the docs, maybe this will work?
```
:class:`amici.amici.ReturnData`
```

https://amici--1817.org.readthedocs.build/en/1817/generated/amici.petab_objective.html#amici.petab_objective.simulate_petab